contracts/UniswapV3Staker.sol

```diff
@@ -292,22 +295,27 @@ contract UniswapV3Staker is
             SafeMath.sub(incentives[incentiveId].totalRewardUnclaimed, reward)
         );
 
-        /* TODO: This will be fixed in https://github.com/omarish/uniswap-v3-staker/issues/38
-        so that collecting and unstaking are two separate functions */
-        try
-            IERC20Minimal(incentives[incentiveId].rewardToken).transfer(
-                params.to,
-                reward
-            )
-        returns (bool) {
-            // It didn't fail
-        } catch {
-            // It failed
-        }
+        rewards[incentives[incentiveId].rewardToken][msg.sender] += reward;
```

Comment:
we'll probably want to use SafeMath here to avoid overflow

Reply:
Done!